### ISSUE TITLE
How to get the best parameter？

### ISSUE THREAD

**Issue by ToneLi** [malllabiisc/EmbedKGQA#6]
Can you supply the script about how to get the best parameters for your model? Includes: learning rate, batch_size, etc

**Comment by IISCAditayTripathi** [comment]
We did the grid based hyper-parameter tuning to get the best models. We will share the best hyper-parameters in few days.

**Comment by justin020025** [comment]
Hi, have you shared it?  If so, where can I find that? Thanks.
